Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/app
```

Comment:
So would it be /usr/src/app or path.join(__dirname + 'usr/src/app') ? Same thing for the other ones?